sdk/workloadmonitor/Microsoft.Azure.Management.WorkloadMonitor/tests/Microsoft.Azure.Management.WorkloadMonitor.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>WorkloadMonitor.Tests</PackageId>
+    <Description>WorkloadMonitor.Tests Class Library</Description>
+    <AssemblyName>WorkloadMonitor.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>
+  </PropertyGroup>
+  
+  <ItemGroup>
```

Comment:
Need to add a section that will copy over your test recording files, here is an example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj#L23

Reply:
got it, added the sessionRecords itemGroup